### ISSUE TITLE
Question regarding armor

### ISSUE THREAD

**Issue by Sunconure11** [Leviathan-Studio/CraftStudioAPI#32]
Does this API have support for animating armor? Say, what if I want a hat that wobbles, does the API support animations created for armor?

**Comment by ZeAmateis** [comment]
Yeah propably if you use the RenderPlayerEvent, by calling all basics to animate an entity, I've not tested that things, but it's a great idea !

I could probably try to animate an armor (or the player literally) and if it's a success, I will make a tutorial on the Wiki !

**Comment by ZeAmateis** [comment]
@Timmypote Work on it actually ! Stay tuned !
